src/linux_disk.py

```diff
@@ -11,7 +11,7 @@
 
 def parse_args():
     parser = ArgumentParser()
-    parser.add_argument('--prefix', default='disk_io')
+    parser.add_argument('--prefix', default='disk')
```

Comment:
I approve this name because we are using such metric name anyway. Also there is not many other "disk" things.